### PR TITLE
Charm V8

### PR DESCRIPTION
**Pull request by SimonRichardson**
The following updates all the references to charm V7, so that they point
to the correct version - V8

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,15 +1,11 @@
-github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
-github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
-github.com/gabriel-samfira/sys v0.0.0-20150608132119-9ddc60d56b51 h1:rUp9t/FbeJM3R3BSYkJfViN3CNQcmk44H20SqkJ/y+k=
-github.com/gabriel-samfira/sys v0.0.0-20150608132119-9ddc60d56b51/go.mod h1:j3qx7fgMceeChTk5ItmRVrWR2ZOTHI9ymSVsxolzC/g=
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
```
Comment (achilleasa) [inline go.sum:1]: Btw, who brings this in? Is it some test?

Comment (SimonRichardson) [reply go.sum:1]: `go mod why` tells me nothing and it's not in the code directly, so I'm actually unsure where spew comes from indirectly.

**Comment by SimonRichardson** [comment]
$$merge$$

**Comment by SimonRichardson** [comment]
https://trello.com/c/vp1aWZig/2854-update-charm-to-v8

**Pull request merged**